github-adapter/src/main/java/be/xplore/githubmetrics/githubadapter/PullRequestsAdapter.java

```diff
@@ -72,4 +88,19 @@ public List<PullRequest> getAllPullRequestsForRepository(Repository repository)
 
         return pullRequests;
     }
+
+    @Override
+    public boolean freshDataCanWait() {
+        return this.rateLimitState.shouldDataWait(ApiRateLimitStatus.GOOD);
```

Comment:
I think all these should be .env configurable

Reply:
done